src/main/resources/standardTestCases/backend.test

```diff
@@ -0,0 +1,17 @@
+//Test against a different functional backend in the future
+
+name: backend
+testFormat: workflowsuccess
+
+files {
+  wdl: hello/hello.wdl
+  inputs: hello/hello.inputs
+  options: backend/backend.options
+}
+
+metadata {
+  workflowName: hello
+  status: Succeeded
+  "calls.hello.hello.executionStatus": Done
+  "calls.hello.hello.backend": "Local"
+}
```

Comment:
I know that @kshakir likes to end files with newlines. I personally couldn't possible comment...